docs/content/integrations/embedded-elt.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Dagster Embedded ELT"
+description: Lightweight ELT with Dagster
+---
+
+# Dagster Embedded ELT
+
+This package provides a framework for building ELT pipelines with Dagster through helpful pre-built assets and resources. It is currently in experimental development, and we'd love to hear your feedback.
+
+This package currently includes a single implementation using <a href="https://slingdata.io">Sling</a> provides a simple way to sync data between databases and file systems.
+
+We plan on adding additional embedded ELT tool integrations in the future.
+
+## Relevant APIs
+
+| Name                                                                        | Description                                                                    |
+| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| <PyObject module="dagster_embedded_elt.sling" object="build_sling_asset" /> | The core Sling asset factory for building syncs                                |
+| <PyObject module="dagster_embedded_elt.sling" object="SlingResource" />     | The Sling Resource used for handing credentials to databases and object stores |
+
+---
+
+## Getting Started
+
+To get started with `dagster-embedded-elt` and Sling, first familiarize yourself with <a href="https://docs.slingdata.io/sling-cli/running-tasks">Sling's documentation</a>
+
+First, setup a <PyObject module="dagster-embedded-elt.sling" object="SlingResource" /> which defines the source and destination credentials. The <PyObject module="dagster-embedded-elt.sling" object="SlingResource" /> requires both a <PyObject module="dagster-embedded-elt.sling" object="SlingSourceConnection" /> and a <PyObject module="dagster-embedded-elt.sling" object="SlingTargetConnection" />.
```

Comment:
You're doing this in the next section (`Setting up a Sling resource`) - why would it also be an instruction here?

Reading through this, I'm not totally sure what the purpose of the `Getting started` section is - is it to outline the steps the user will be taking in the rest of the guide?